src/main_app/app_routes/admin_routes/coordinators.py

```
@@ -26,8 +26,8 @@ def _coordinators_dashboard() -> str:
     """Render the coordinator management dashboard."""
     try:
         coordinators = admin_service.list_coordinators()
-    except Exception:  # pragma: no cover - defensive guard
-        logger.exception("Unable to list coordinators.")
+    except Exception as e:  # pragma: no cover - defensive guard
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)